ecs-deploy

```diff
@@ -407,6 +409,33 @@ function waitForGreenDeployment {
   fi
 }
 
+function waitForDeplomentCompletion {
+  echo "Waiting for deployment completion until service is stable"
+
+  WAIT="true"
+  DEPLOYMENT_SUCCESS="false"
+  
+  while [ "${WAIT}" != "false" ]
+  do
+
+    if NEW_DEPLOYMENT=$($AWS_ECS wait services-stable --cluster $CLUSTER --services $SERVICE); then
```

Comment:
As we talked in slack, we have to handle the exit code here in case of non-zero values so the script doesn't quit abruptly. Thanks for catching this!